test/e2e/operations/iterateDirectory.js

```
@@ -14,6 +14,11 @@ class FileIterator extends Readable {
     const directory = resolve(basePath, dirName)
 
     readdir(directory, (e, files) => {
+      if (e) {
```

what is the purpose of this?

---

That's just for tests. Without forwarding the error, it was swallowed